lib/cdo/languages.rb

```diff
@@ -32,6 +32,10 @@ def self.table
     table.select(:unique_language_s, :locale_s).where("locale_s = '#{locale}'").first[:unique_language_s]
   end
 
+  cached def self.get_native_name_by_locale(locale)
+    table.select(:native_name_s, :locale_s).where("locale_s = '#{locale}'").to_a
```

Comment:
does this method error if we pass in an invalid `locale`?

Reply:
no, it just returns empty.